agent/config/config.go

```diff
@@ -155,6 +155,11 @@ type Nomad struct {
 
 	// SkipVerify enables or disables SSL verification.
 	SkipVerify bool `hcl:"skip_verify,optional"`
+
+	// BlockQueryWaitTime controls how long Nomad API requests supporting blocking queries
+	// are held open. Defaults to 5m.
+	BlockQueryWaitTime    time.Duration
```

Comment:
As a general note regarding the option name choice: I waffled between a few options before settling on this. Primarily motivated by this bit of similar config in the Nomad codebase (though related to a Consul API client setting in that case 🤷🏻): https://github.com/hashicorp/nomad/blob/v1.7.0-beta.1/client/config/config.go#L380-L387

(Happy to call it whatever though 😄 )

Reply:
I think the name looks good to me 👍